openmls/src/group/mls_group/mod.rs

```diff
@@ -239,6 +239,12 @@ impl MlsGroup {
         self.proposal_store.proposals()
     }
 
+    /// Inserts the given proposal into the proposal store.
+    #[cfg(test)]
+    pub fn add_pending_proposal(&mut self, queued_proposal: QueuedProposal) {
```

Comment:
We already have the function `store_pending_proposal()` in `processing.rs` for that.

Reply:
Good point, I totally missed that.